src/main/java/se/sundsvall/supportmanagement/service/RevisionService.java

```diff
@@ -76,15 +76,18 @@ public class RevisionService {
 	 */
 	public RevisionResult createErrandRevision(ErrandEntity entity) {
 
+		LOG.info("=== findFirstByEntityIdOrderByVersionDesc ===");
```

Comment:
Should the log entries in this method stay? If so, maybe add more useful information?